jdk25.0.4+1 HotSpotAOTCacheImpl.java fails to compile

https://openj9-jenkins.osuosl.org/job/Pipeline-OpenJDK25-Acceptance/24/

https://openj9-jenkins.osuosl.org/job/Build_JDK25_aarch64_mac_OpenJDK25/23
```
10:18:49  /Users/jenkins/workspace/Build_JDK25_aarch64_mac_OpenJDK25/src/jdk.management/share/classes/com/sun/management/internal/HotSpotAOTCacheImpl.java:30: error: cannot access VMManagement
10:18:49  import sun.management.VMManagement;
10:18:49                       ^
10:18:49    bad source file: /Users/jenkins/workspace/Build_JDK25_aarch64_mac_OpenJDK25/build/macosx-aarch64-server-release/support/j9jcl/java.management/share/classes/sun/management/VMManagement.java
10:18:49      file does not contain class sun.management.VMManagement
10:18:49      Please remove or make sure it appears in the correct subdirectory of the sourcepath.
10:18:49  /Users/jenkins/workspace/Build_JDK25_aarch64_mac_OpenJDK25/src/jdk.management/share/classes/com/sun/management/internal/HotSpotAOTCacheImpl.java:40: error: cannot find symbol
10:18:49      private final VMManagement jvm;
10:18:49                    ^
10:18:49    symbol:   class VMManagement
10:18:49    location: class HotSpotAOTCacheImpl
10:18:49  /Users/jenkins/workspace/Build_JDK25_aarch64_mac_OpenJDK25/src/jdk.management/share/classes/com/sun/management/internal/HotSpotAOTCacheImpl.java:44: error: cannot find symbol
10:18:49      HotSpotAOTCacheImpl(VMManagement vm) {
10:18:49                          ^
10:18:49    symbol:   class VMManagement
10:18:49    location: class HotSpotAOTCacheImpl
10:18:49  3 errors
```

--- a/jcl/src/jdk.management/share/classes/com/sun/management/internal/HotSpotAOTCacheImpl.java
+++ b/jcl/src/jdk.management/share/classes/com/sun/management/internal/HotSpotAOTCacheImpl.java
@@ -1,4 +1,4 @@
-/*[INCLUDE-IF JAVA_SPEC_VERSION >= 26]*/
+/*[INCLUDE-IF JAVA_SPEC_VERSION >= 25]*/
 /*
  * Copyright IBM Corp. and others 2025
  *

